lib/workers/repository/update/pr/body/index.ts

```diff
@@ -85,6 +89,16 @@ export async function getPrBody(
     prBody = template.compile(prBodyTemplate, content, false);
     prBody = prBody.trim();
     prBody = prBody.replace(regEx(/\n\n\n+/g), '\n\n');
+    const debugData = updatePrDebugData(prBodyConfig?.debugData);
+    const prDebugData64 = toBase64(JSON.stringify(debugData));
+    if (prBodyConfig?.debugData) {
+      prBody = prBody.replace(
```

Comment:
I think this is incorrect, since `prBodyTemplate` doesn't contain `<!--renovate-debug:...-->`.
Simply appending `<!--renovate-debug:${prDebugData64}-->` to the `prBody` should do the trick.